### PR TITLE
HTML Add Media: style #3556

### PR DESCRIPTION
**Pull request by kitttrick**
### Description

- Added an output header
- Added an image to demonstrate an example of the code
- Closes Issue #3336 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: media/style-attribute-example.PNG
Comment (SSwiniarski) [inline media/style-attribute-example.PNG:1]: This does not look like the example style. Can we do full size on a white background?

File: content/html/concepts/attributes/terms/style/style.md [hunk 1]
```diff
@@ -37,3 +37,8 @@ In the example below, the [`<p>`](https://www.codecademy.com/resources/docs/html
   Here's some text that is being styled using the style attribute.
 </p>
 ```
+## Output
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/style/style.md:40]:
```suggestion
```
We do not need a header for the output

File: media/style-attribute-example.PNG
Comment (SSwiniarski) [inline media/style-attribute-example.PNG:1]: Also, can we have ".png" in lower case?

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@kitttrick I've done a first review and left some comments.

**Review by kitttrick — COMMENTED** [review]
I believe I've now made all requested changes

File: media/style-attribute-example.PNG
Comment (SSwiniarski) [inline media/style-attribute-example.PNG:1]: we need to get rid of this file

File: content/html/concepts/attributes/terms/style/style.md [hunk 2]
```diff
@@ -37,3 +37,7 @@ In the example below, the [`<p>`](https://www.codecademy.com/resources/docs/html
   Here's some text that is being styled using the style attribute.
 </p>
 ```
+
+The example above results in the following output:
+
+![Style Attribute Example](https://raw.githubusercontent.com/Codecademy/docs/main/media/media/style-attribute-example.PNG)
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/style/style.md:43]:
```suggestion
![Style Attribute Example](https://raw.githubusercontent.com/Codecademy/docs/main/media/media/style-attribute-example.png)
```

File: media/style-attribute-example.png
Comment (SSwiniarski) [inline media/style-attribute-example.png:1]: Can we zoom in on this so the text is legible.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@kitttrick some more comments

**Comment by SSwiniarski** [comment]
@kitttrick You still need to remove that file and make the image text legible. If there's no movement on this PR in a few days, we'll have to close it and unassign the issue.

**Comment by SSwiniarski** [comment]
Closing because PR was resubmitted.

**Pull request closed without merging**